assets/walls/japanese/japanese_wall_bottom_plain.json

```diff
@@ -2,6 +2,6 @@
     "name" : "japanese_wall_bottom_plain",
     "type" : "wall",
     "filepath" : "assets/walls/japanese/japanese_wall_bottom_plain.nbt",
-    "origin" : [1, 1, 0],
+    "origin" : [1, -1, 0],
```

Comment:
The origin is supposed to be relative to the nbt file. That means, since the wall is at y=1, its origin is -1 relative to the nbt file's y=0. Messed me up for a sec.